.changeset/sixty-shoes-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-react': patch
+---
+
+Make PodExecTerminal UI OptIn
```

Comment:
This might not pass the spell checker, but I think we can give a more descriptive release note:
```suggestion
**BREAKING** The pod exec terminal is now disabled by default since there are several scenarios where it is known not to work. It can be re-enabled at your own risk by setting the config parameter `kubernetes.podExecTerminal.enabled` to `true`.
```